docs/pages/architecture/proxy-peering.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Proxy Peering
+description: How Teleport implements more efficient networking with proxy peering.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="10.0"
+>
+  Proxy peering is available starting from Teleport `10.0`.
+</Details>
+
+## Overview
+
+Proxy peering is a type of tunnel strategy configurable in Teleport. A tunnel
+strategy defines the behavior for agent reverse tunnel connections as well as
+how client are connected to those reverse tunnels. A Teleport Cluster's tunnel
+strategy is configurable in the Teleport Auth service.
+
+Proxy peering enables an agent to be reachable without it connecting to every
+proxy instance. This allows proxies to scale horizontally without increasing
```

Comment:
```suggestion
Proxy Service instance. This allows Proxy Service instances to scale horizontally without increasing
```
The term "proxy" comes up in various contexts in our docs, so we should be more precise here so readers can understand our architecture more easily.